dplython/dplython.py

```diff
@@ -434,13 +434,17 @@ def addColumns(df):
 @ApplyToDataframe
 def group_by(*args, **kwargs):
   def GroupDF(df):
-    if args and max([len(arg.todo) for arg in args]) > 1:
-      raise ValueError(
-        "Expressions not allowed as positional args. Use keyword args.")
-    group_columns = [arg.name for arg in args]
-    if kwargs:
-      group_columns.extend(kwargs.keys())
-      df = df >> mutate(**kwargs)
+    group_columns = kwargs.keys()
```

Comment:
I think this is breaking your code in Python 3. Try
`group_columns = list(kwargs.keys())`
instead